### PR TITLE
[Bridges] fix deleting variable in Bridges.Variable.VectorizeBridge

### PR DESCRIPTION
**Pull request by odow**
One issue from #2384.

It seems simpler to throw here than to carry around a bunch of extra state about which constants we need to modify.

### CODE REVIEW

File: src/Bridges/Variable/bridges/vectorize.jl [hunk 1]
```diff
@@ -98,10 +98,14 @@ function MOI.get(
     return [bridge.vector_constraint]
 end
 
-# References
 function MOI.delete(model::MOI.ModelLike, bridge::VectorizeBridge)
-    MOI.delete(model, bridge.variable)
-    return
+    # It isn't safe to delete the variable because the constant may appear in
+    # other parts of the model (like the objective, or right-hand side sets).     Therefore, we don't implement
```
Comment (odow) [inline src/Bridges/Variable/bridges/vectorize.jl:103]:
```suggestion
    # other parts of the model (like the objective, or right-hand side sets).
```

**Comment by odow** [comment]
@blegat you might want to take a look here. I don't really understand all of the different levels of indirection.

**Comment by blegat** [comment]
What's wrong with `ConstraintPrimal` ?

**Comment by odow** [comment]
> What's wrong with ConstraintPrimal ?

I couldn't figure it out yet. It gets the wrong answer, but the layers of debugging are many...

But I'll remove for now. The delete fix is correct.

**Pull request merged**